packages/python/readme_metrics/MetricsApiConfig.py

```diff
@@ -69,12 +70,16 @@ def __init__(
 
         Args:
             api_key (str): Your ReadMe API key
-            grouping_function ([type]]): Grouping function to construct an identity
+            grouping_function ([type] or str]): Grouping function to construct an identity
```

Comment:
This change was required to make this more friendly for Django configuration. A user will set up their `MetricsApiConfig` in their Django app's `settings.py` file, but I don't think it's generally a good idea to import other modules from their codebase into `settings.py` (this could cause circular dependencies).